README.md

```diff
@@ -1,7 +1,7 @@
-# Translator [![Build Status](https://travis-ci.org/Sage/translator.svg?branch=master)](https://travis-ci.org/Sage/translator) [![Maintainability](https://api.codeclimate.com/v1/badges/f71fb2039dd7d50eb90d/maintainability)](https://codeclimate.com/github/Sage/translator/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/f71fb2039dd7d50eb90d/test_coverage)](https://codeclimate.com/github/Sage/translator/test_coverage) [![Documentation Status](http://inch-ci.org/github/Sage/translator.svg?branch=master)](http://inch-ci.org/github/Sage/translator) [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/Sage/translator/blob/master/LICENSE) [![Dependency Status](https://gemnasium.com/badges/github.com/Sage/translator.svg)](https://gemnasium.com/github.com/Sage/translator)
+# I18n Yaml Editor [![Build Status](https://travis-ci.org/Sage/i18n_yaml_editor.svg?branch=master)](https://travis-ci.org/Sage/i18n_yaml_editor) [![Maintainability](https://api.codeclimate.com/v1/badges/f71fb2039dd7d50eb90d/maintainability)](https://codeclimate.com/github/Sage/i18n_yaml_editor/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/f71fb2039dd7d50eb90d/test_coverage)](https://codeclimate.com/github/Sage/i18n_yaml_editor/test_coverage) [![Documentation Status](http://inch-ci.org/github/Sage/i18n_yaml_editor.svg?branch=master)](http://inch-ci.org/github/Sage/i18n_yaml_editor) [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/Sage/i18n_yaml_editor/blob/master/LICENSE) [![Dependency Status](https://gemnasium.com/badges/github.com/Sage/i18n_yaml_editor.svg)](https://gemnasium.com/github.com/Sage/i18n_yaml_editor)
```

Comment:
Should we keep the repo name as `translator` for a while before we rename so we have chance to switch over usage from GitHub to RubyGems?

Reply:
To me, renaming should include repo and all integrations as well. I'm aware there is some extra effort involved, but I don't see any benefits in keeping the old name anywhere since this doesn't seem to be used outside Sage today.

Reply:
In which case let's go for it.